### PR TITLE
Update the company address across the site

### PR DESCRIPTION
**Pull request by anthonydillon**
> Please note that you may need to update copyright notices to:
Canonical Limited, Registered in Isle of Man, Company number 110334C
Registered Office:  2nd Floor, Clarendon House, Victoria Street, Douglas IM1 2LN, Isle of Man

## Done

- Update the registered address across the website
- Updated the company number

## QA

- Check the code

### CODE REVIEW

File: templates/legal/data-privacy/cube-beta.md [hunk 1]
```diff
@@ -78,10 +78,7 @@ We hope that we can resolve any query or concern you raise about our use of your
 <div style="margin: 2rem;">
   <p>
     Legal, Canonical<br />
-    The Office Group, St Dunstans House, <br />
-    4th floor, 201 Borough High St, <br />
-    London SE1 1JA,<br />
-    United Kingdom
+    2nd Floor, Clarendon House, Victoria Street, Douglas IM1 2LN, Isle of Man
```
Comment (petesfrench) [inline templates/legal/data-privacy/cube-beta.md:81]:
```suggestion
    2nd Floor, Clarendon House, <br/>
    Victoria Street, <br/>
    Douglas IM1 2LN, <br/> 
    Isle of Man
```
This address was previously using breaks

File: templates/legal/data-privacy/esxi.md [hunk 2]
```diff
@@ -81,10 +81,7 @@ We hope that we can resolve any query or concern you raise about our use of your
 
 <div style="margin:2rem;">
 Legal, Canonical<br />
-The Office Group, St Dunstans House, <br />
-4th floor, 201 Borough High St, <br />
-London SE1 1JA, <br />
-United Kingdom
+2nd Floor, Clarendon House, Victoria Street, Douglas IM1 2LN, Isle of Man
```
Comment (petesfrench) [inline templates/legal/data-privacy/esxi.md:84]:
```suggestion
    2nd Floor, Clarendon House, <br/>
    Victoria Street, <br/>
    Douglas IM1 2LN, <br/> 
    Isle of Man
```
same here with breaks

File: templates/legal/data-privacy/recruitment.html [hunk 3]
```diff
@@ -20,7 +20,7 @@ <h2 id="introduction">1. Introduction</h2>
 
       <p>We are Canonical Group Limited and are the “data controller” for the purposes of applicable data protection regulations relevant to your personal information. This means that we are responsible for determining how we hold and use your personal information.</p>
 
-      <p>Canonical is a limited company registered in England and Wales with registered number 06870835. Our registered office is 5 New Street Square, London EC4A 3TW.</p>
+      <p>Canonical is a limited company registered in England and Wales with registered number 0110334C. Our registered office is 5 New Street Square, London EC4A 3TW.</p>
```
Comment (petesfrench) [inline templates/legal/data-privacy/recruitment.html:23]: How can I check this is correct?

**Review by petesfrench — APPROVED** [review]
LGTM, there are quite a few cases where line breaks are no longer being used in the address. I have highlighted a couple, but after finding more I wasn't sure if this was intentional, so stopped commenting on them.

**Comment by anthonydillon** [comment]
Yeah I started to get lazy then thought if they all used the same pattern (flat) we could use a variable. But good spot.

File: templates/legal/data-privacy/events.md [hunk 4]
```diff
@@ -107,10 +107,7 @@ We hope that we can resolve any query or concern you raise about our use of your
 
 <div style="margin:2rem;">
 Legal, Canonical<br />
-The Office Group, St Dunstans House, <br />
-4th floor, 201 Borough High St, <br />
-London SE1 1JA,  <br />
-United Kingdom
+2nd Floor, Clarendon House, Victoria Street, Douglas IM1 2LN, Isle of Man
```
Comment (petesfrench) [inline templates/legal/data-privacy/events.md:110]:
```suggestion
2nd Floor, Clarendon House, <br/>Victoria Street, <br/>Douglas IM1 2LN, <br/>Isle of Man
```

File: templates/legal/data-privacy/index.html [hunk 5]
```diff
@@ -292,10 +292,7 @@ <h2 id="contact">Contact</h2>
       <div style="margin: 2rem;">
         <p>
           Legal, Canonical<br />
-          The Office Group, St Dunstans House, <br />
-          4th floor, 201 Borough High St, <br />
-          London SE1 1JA, <br />
-          United Kingdom
+          2nd Floor, Clarendon House, Victoria Street, Douglas IM1 2LN, Isle of Man
```
Comment (petesfrench) [inline templates/legal/data-privacy/index.html:295]:
```suggestion
          2nd Floor, Clarendon House, <br/>Victoria Street, <br/>Douglas IM1 2LN, <br/>Isle of Man
```

File: templates/legal/data-privacy/online-competitions.md [hunk 6]
```diff
@@ -105,10 +105,7 @@ We hope that we can resolve any query or concern you raise about our use of your
 <div style="margin: 2rem;">
   <p>
     Legal, Canonical<br />
-    The Office Group, St Dunstans House, <br />
-    4th floor, 201 Borough High St, <br />
-    London SE1 1JA,<br />
-    United Kingdom
+    2nd Floor, Clarendon House, Victoria Street, Douglas IM1 2LN, Isle of Man
```
Comment (petesfrench) [inline templates/legal/data-privacy/online-competitions.md:108]:
```suggestion
    2nd Floor, Clarendon House, <br/>Victoria Street, <br/>Douglas IM1 2LN, <br/>Isle of Man
```

File: templates/legal/data-privacy/online-purchase.md [hunk 7]
```diff
@@ -113,10 +117,7 @@ We hope that we can resolve any query or concern you raise about our use of your
 
 <div style="margin:2rem;">
 Legal, Canonical<br />
-The Office Group, St Dunstans House, <br />
-4th floor, 201 Borough High St, <br />
-London SE1 1JA,<br />
-United Kingdom
+2nd Floor, Clarendon House, Victoria Street, Douglas IM1 2LN, Isle of Man
```
Comment (petesfrench) [inline templates/legal/data-privacy/online-purchase.md:120]:
```suggestion
2nd Floor, Clarendon House, <br/>Victoria Street, <br/>Douglas IM1 2LN, <br/>Isle of Man
```

File: templates/legal/data-privacy/partner-portal.md [hunk 8]
```diff
@@ -126,10 +125,7 @@ We hope that we can resolve any query or concern you raise about our use of your
 
 <div style="margin:2rem;">
 Legal, Canonical<br />
-The Office Group, St Dunstans House, <br />
-4th floor, 201 Borough High St, <br />
-London SE1 1JA, <br />
-United Kingdom
+2nd Floor, Clarendon House, Victoria Street, Douglas IM1 2LN, Isle of Man
```
Comment (petesfrench) [inline templates/legal/data-privacy/partner-portal.md:128]:
```suggestion
2nd Floor, Clarendon House, <br/>Victoria Street, <br/>Douglas IM1 2LN, <br/>Isle of Man
```

File: templates/legal/data-privacy/snap-store.md [hunk 9]
```diff
@@ -114,10 +115,7 @@ We hope that we can resolve any query or concern you raise about our use of your
 
 <div style="margin:2rem;">
 Legal, Canonical<br />
-The Office Group, St Dunstans House, <br />
-4th floor, 201 Borough High St, <br />
-London SE1 1JA,<br />
-United Kingdom
+2nd Floor, Clarendon House, Victoria Street, Douglas IM1 2LN, Isle of Man
```
Comment (petesfrench) [inline templates/legal/data-privacy/snap-store.md:118]:
```suggestion
2nd Floor, Clarendon House, <br/>Victoria Street, <br/>Douglas IM1 2LN, <br/>Isle of Man
```

File: templates/legal/data-privacy/snapcraft-nps.md [hunk 10]
```diff
@@ -77,10 +78,7 @@ We hope that we can resolve any query or concern you raise about our use of your
 
 <div style="margin:2rem;">
 Legal, Canonical<br />
-The Office Group, St Dunstans House, <br />
-4th floor, 201 Borough High St, <br />
-London SE1 1JA,<br />
-United Kingdom
+2nd Floor, Clarendon House, Victoria Street, Douglas IM1 2LN, Isle of Man
```
Comment (petesfrench) [inline templates/legal/data-privacy/snapcraft-nps.md:81]:
```suggestion
2nd Floor, Clarendon House, <br/>Victoria Street, <br/>Douglas IM1 2LN, <br/>Isle of Man
```

File: templates/legal/data-privacy/survey.md [hunk 11]
```diff
@@ -81,10 +81,7 @@ We hope that we can resolve any query or concern you raise about our use of your
 
 <div style="margin:2rem;">
 Legal, Canonical<br />
-The Office Group<br />
-4th floor, 201 Borough High St,<br />
-London SE1 1JA,<br />
-United Kingdom
+2nd Floor, Clarendon House, Victoria Street, Douglas IM1 2LN, Isle of Man
```
Comment (petesfrench) [inline templates/legal/data-privacy/survey.md:84]:
```suggestion
2nd Floor, Clarendon House, <br/>Victoria Street, <br/>Douglas IM1 2LN, <br/>Isle of Man
```

File: templates/legal/data-privacy/user-survey.md [hunk 12]
```diff
@@ -78,10 +79,7 @@ We hope that we can resolve any query or concern you raise about our use of your
 
 <div style="margin:2rem;">
 Legal, Canonical<br />
-The Office Group, St Dunstans House, <br />
-4th floor, 201 Borough High St, <br />
-London SE1 1JA,<br />
-United Kingdom
+2nd Floor, Clarendon House, Victoria Street, Douglas IM1 2LN, Isle of Man
```
Comment (petesfrench) [inline templates/legal/data-privacy/user-survey.md:82]:
```suggestion
2nd Floor, Clarendon House, <br/>Victoria Street, <br/>Douglas IM1 2LN, <br/>Isle of Man
```

File: templates/legal/terms-and-policies/cube-terms.html [hunk 13]
```diff
@@ -165,9 +165,7 @@ <h2>9. General</h2>
             </p>
             <p>
                 Canonical Group Limited,<br>
-                The Office Group, St Dunstans House, <br />
-                4th floor, 201 Borough High St, <br />
-                London SE1 1JA<br>
+                2nd Floor, Clarendon House, Victoria Street, Douglas IM1 2LN, Isle of Man
```
Comment (petesfrench) [inline templates/legal/terms-and-policies/cube-terms.html:168]:
```suggestion
                2nd Floor, Clarendon House, <br/>Victoria Street, <br/>Douglas IM1 2LN, <br/>Isle of Man
```

File: templates/legal/terms.html [hunk 14]
```diff
@@ -43,7 +43,7 @@ <h2 id="terms-disclaimer">Disclaimer</h2>
     <div class="col-4">
       <div class="p-card">
         <h3>Registered office</h3>
-        <p>The Office Group, St Dunstans House, <br />4th floor, 201 Borough High St, <br />London SE1 1JA</p>
+        <p>2nd Floor, Clarendon House, Victoria Street, Douglas IM1 2LN, Isle of Man</p>
```
Comment (petesfrench) [inline templates/legal/terms.html:46]:
```suggestion
        <p>2nd Floor, Clarendon House, <br/>Victoria Street, <br/>Douglas IM1 2LN, <br/>Isle of Man</p>
```

**Comment by minkyngkm** [comment]
I've added some breaks as you suggested @petesfrench. Could you take a look again? Thanks Pete! :D

**Review by petesfrench — APPROVED** [review]
LGTM

**Pull request merged**